Migration.md

```diff
@@ -0,0 +1,40 @@
+# Cloud Migration
+
+Die Entscheidung für eine Cloud-Strategie kann diverse Ursachen und Beweggründe haben.
+Das Warum ist für die erfolgreiche Umsetzung einer Cloud-Strategie nicht entscheidend, viel mehr sind es die Rahmenbedingungen innerhalb der die Umsetzung erfolgt.
+
+## Cloud-Strategie
+
+Die Umsetzung einer Cloud-Strategie findet meist in folgendem Spannungsfeld statt:
+
+![](tension-triangle.svg)
+
+* **Regulatorik** - Hierunter fallen alle unabdingbaren Rahmenbedingungen wie Datenschutz, Compliance, Rechtliche- oder Unternehmensvorgaben ("Governance")
+* **Produkt** - Fach-Teams, deren primäre Aufgabe das Liefern von Business-Value ist
+* **Plattform** - Technische wie rganisatorische Basis auf der die Fach-Teams Arbeiten
+
+Eine Cloud-Migration erfolgt zumeist während des laufenden Geschäftsbetriebs. Neben dem Einführen neuer Technologien und Methodiken in der IT, ist das Bewusstsein um die Änderungen die der Einsatz der Cloud idealerweise mit sich bringt, auch in den Fachbereichen absolut notwendig.
+
+**Es wird generell davon ausgegangen, dass eine Cloud-Strategie nicht nur als eine Änderung im Hosting bedeutet, sondern gleichermaßen eine Transformation in eine DevOps-Organisationsform vollzogen wird, also Fach-Teams eigenständig und selbstverantwortlich lieferfähig gemacht werden sollen.** _(vgl. [Organisatonsform](Organisation.md))_
+
+## Rahmenbedingungen
+
+Die folgenden Umstände werden als förderlich in der Umsetzung einer Cloud-Strategie angesehen.
+Diese können grob in organisatorische wie technische Faktoren unterteilt werden, wobei die Abgrenzung nicht immer trennscharf ist.
+
```

Comment:
diese 3 Listen sind noch in Arbeit, da fehlt noch einiges, daher bitte noch ignorieren. Ich geb' Bescheid, wenn sie vollständig sind.